TwitchPlaysAssembly/Src/Commands/BombCommands.cs

```diff
@@ -108,6 +108,59 @@ public static IEnumerator Explode(TwitchBomb bomb, string user, bool isWhisper)
 	[Command(@"(explode|detonate|kapow)", AccessLevel.Mod, AccessLevel.Mod)]
 	public static IEnumerator Explode(TwitchBomb bomb) => bomb.DelayBombExplosionCoroutine();
 
+	#region Voting
+	/// <name>Start a vote</name>
+	/// <syntax>vote [action]</syntax>
+	/// <summary>Starts a vote about doing an action</summary>
+	[Command(@"vote (explode|detonate|kapow)")]
+	public static void VoteStart(TwitchBomb bomb, string user, [Group(1)] bool Detonation) => Votes.StartVote(bomb, user, Detonation ? "Detonation" : "");
+
+	/// <name>Vote</name>
+	/// <syntax>vote [choice]</syntax>
+	/// <summary>Vote with yes or no</summary>
+	[Command(@"vote (yes|voteyea)|(no|votenay)")]
+	public static void Vote(string user, [Group(1)] bool yesVote) => Votes.Vote(user, yesVote);
+
+	/// <name>Remove vote</name>
+	/// <syntax>vote remove</syntax>
+	/// <summary>Removes the vote of a user</summary>
+	[Command(@"vote remove")]
+	public static void RemoveVote(string user) => Votes.RemoveVote(user);
+
+	/// <name>Cancel vote</name>
+	/// <syntax>vote cancel</syntax>
+	/// <summary>Cancels a voting process</summary>
+	/// <restriction>Mod</restriction>
+	[Command(@"vote cancel", AccessLevel.Mod)]
+	public static void CancelVote()
+	{
+		if (!Votes._global && !Votes._going)
+		{
+			IRCConnection.SendMessage("There is no voting currently in progress.");
+			return;
+		}
+		Votes._global = false;
+		Votes.Clear();
+		IRCConnection.SendMessage("Voting got canceled");
+	}
+
+	/// <name>Force-end vote</name>
+	/// <syntax>vote forceend</syntax>
+	/// <summary>Skips the countdown of the voting process</summary>
+	/// <restriction>Mod</restriction>
+	[Command(@"vote forceend", AccessLevel.Mod)]
+	public static void ForceEndVote()
+	{
+		if (!Votes._global && !Votes._going)
+		{
+			IRCConnection.SendMessage("There is no voting currently in progress.");
+			return;
+		}
+		IRCConnection.SendMessage("Force-ending vote");
+		Votes.OnTick(null, null);
+	}
```

Comment:
You don't need to define the exact same commands again since they're already defined in the global commands.

Reply:
Yeah I did it because I think it’s better if we have a !bomb vote ‹something› and a !vote ‹something› since we have separate ones for starting a vote

Reply:
Thinking about it, I don't think we should even have separate bomb command votes and global votes. It makes it more confusing for users to remember what's a bomb vote and what's a global vote. Like, at least to me, detonating the bomb isn't specific to a bomb since you can detonate any bomb and have the same effect.

Reply:
Ah alright

Reply:
Why did you not do this to Vote and Remove vote?

Reply:
I tought it's better there to call the Votes.Vote() and Votes.RemoveVote() instead of calling a method which calls them but I can do them to if you want

Reply:
Well I'm just trying to reduce duplicate code so that if the code required to vote changes in the future both places don't need to change.

Reply:
It won't change (Or at least I don't see it change now)

Reply:
Like I said, **if** it changes in the future both places would need to change. That's why I recommended it for the other two commands and the same applies for these two commands.

Reply:
ah I see I just misunderstood you

Reply:
Alright done